content/grants/slam.md

```diff
@@ -0,0 +1,108 @@
+
+- Could Slam issue a default gora to all players in case they don't have any yet? Using crow for this looks promising.
+
+- What implications would a progression system have for a gora's value? If you own a gora with a high win count, and some other future app wants to use that info for something interesting, should that consideration impact the design of the progression system?
+
+- Is it worth it to try and prevent players from cheating? Likely not, but testing the waters here could help the Urbit community discover new mechanisms for sources of truth in decentralized apps.
+
+## Implementation Plan
+
+1. Develop the game in Javascript using the Kaboom.js library.
+
+- I don’t have a ton of experience with JavaScript, but I’m confident that this portion of development will go pretty smoothly. Kaboom seems really easy to use from the little I’ve messed with it, and there’s plenty of JavaScript documentation everywhere.
+
+2. Write the hoon backend to gather data for Slam from %pals and %gora.
+
+- While I have experience writing gall agents, Slam will be more complicated with its inter-app functionality.
+
+- Future improvements/stretch goals to Slam notwithstanding, the primary goal of the hoon agent in this iteration will be the gathering of gora and pal data and the surfacing of that data to the Kaboom.js implementation of the game.
+
+3. Integrate the backend with the frontend.
+
+4. Future/stretch goals:
+
+- %gora have unique properties based on their data
+- multiple AI for computer-controlled combatants
+- score sharing modality (possibly the issuance of a gora from a loser to a winner)
+- %gora stat improvement paths
+- 'arenas' for battle w/ prescribed rules and a 3rd party arbiter of victory for direct p2p play, confidence in scores
+
+Though I will likely work on the first three points concurrently, naturally allowing their designs to mesh and influence each other, it seems unlikely to me that I’ll be able to complete the entire project within the 2-3 month time period that would be appropriate for a single milestone.
+
+I have therefore separated them out into two separate milestones that cover, roughly, bullets one and two+three, respectively.
+
+## Champion
+
+I’ll be working with ~rabsef under a tea towel as a Champion. Rabsef has, with Quartus, produced a lot of software that is live on the network today and I am confident that he will be able to answer the questions I may run in to while writing Slam; things along the lines of: “What does the workflow look like for developing gall agents that require other apps? How do I install apps on a fake ship? etc.” Quartus also produced %gora so I’ll have a direct connection to the %gora development team, should I need any support working with their system.
```

Comment:
```suggestion
I’ll be working with ~rabsef-bicrym as a champion. Rabsef has, with Quartus, produced a lot of software that is live on the network today and I am confident that he will be able to answer the questions I may run in to while writing Slam; things along the lines of: “What does the workflow look like for developing gall agents that require other apps? How do I install apps on a fake ship? etc.” Quartus also produced %gora so I’ll have a direct connection to the %gora development team, should I need any support working with their system.
```

Reply:
We could link here: [~dalten + Quartus github](https://github.com/dalten-collective)